cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/views/tool/index.js

```diff
@@ -121,10 +126,8 @@ export default function Tool() {
       <SectionTip
         label="Resource"
         text={
-          <>
```

Comment:
This isn't what I meant by removing the fragment. The way you're passing content to the `text` prop _requires_ the use of a fragment right now, but that's the wrong approach and not idiomatic (i.e. the commonly accepted way to phrase things).

If you need to pass nested components (like HTML content more complex than plain text) to a component, you do it by nesting child components inside of it. So instead of this:

```jsx
<SectionTip
  label="Resource"
  text={
    <>
      You are charged interest...
      <Link to="/resource_url">Here's a link to the resource</Link>
    </>
  }
/>
```

You would write it like this:

```jsx
<SectionTIp label="Resource">
  You are charged interest...
  <Link to="/resource_url">Here's a link to the resource</Link>
</SectionTip>
```

In order to make that work, you need to find the `SectionTip` component and make it use `props.children` instead of `props.text`. See https://reactjs.org/docs/composition-vs-inheritance.html#containment.